ddl/ddl.go

```diff
@@ -698,12 +779,16 @@ func (d *ddl) Start(ctxPool *pools.ResourcePool) error {
 		go d.startCleanDeadTableLock()
 	}
 
-	// If tidb_enable_ddl is true, we need campaign owner and do DDL job.
+	// If tidb_enable_ddl is true, we need campaign owner and do DDL jobs. Besides, we also can do backfill jobs.
 	// Otherwise, we needn't do that.
 	if config.GetGlobalConfig().Instance.TiDBEnableDDL.Load() {
 		if err := d.EnableDDL(); err != nil {
 			return err
 		}
+
+		if err := d.prepareBackfillWorkers(); err != nil {
+			return err
+		}
```

Comment:
If a TiDB instance started with "run-ddl = false", it cannot handle the backfill jobs even if a user turn "run-ddl" on. Is it expected?

Reply:
At present, I think it is expected. Here is dealing with DDL operation, which is the same as DDL owner's behavior of dealing with DDL operation, which I think is understandable